third_party/amd/lib/TritonAMDGPUToLLVM/BuiltinFuncToLLVM.cpp

```diff
@@ -49,9 +49,15 @@ class CallOpConversion : public mlir::RewritePattern {
                mlir::LLVM::AMD::Predicated_Load_NT) != llvm::StringRef::npos;
   }
 
+  bool isPredicatedStoreNT(LLVM::CallOp callOp) const {
+    return callOp.getCallee().value().find(
+               mlir::LLVM::AMD::Predicated_Store_NT) != llvm::StringRef::npos;
+  }
+
   bool isPredicatedStore(LLVM::CallOp callOp) const {
     return callOp.getCallee().value().find(mlir::LLVM::AMD::Predicated_Store) !=
-           llvm::StringRef::npos;
+               llvm::StringRef::npos ||
+           isPredicatedStoreNT(callOp);
```

Comment:
Can you double check if this is necessary? Since "Predicated_store" is also included in "Predicated_store_NT", it can find is in both NT and non-NT cases?